lua/nvim-lsp-installer/servers/lemminx/init.lua

```diff
@@ -31,7 +31,7 @@ return function(name, root_dir)
                 end
             end,
             context.set(function(ctx)
-                ctx.requested_server_version = coalesce(ctx.requested_server_version, "0.18.0-400")
+                ctx.requested_server_version = coalesce(ctx.requested_server_version, "0.18.1-451")
```

Comment:
I did some investigation and it seems like they also distribute the binary through https://download.jboss.org/jbosstools/vscode/snapshots/lemminx-binary, where there's a convenient "LATEST" entry which we could default to. What about:

```suggestion
                ctx.requested_server_version = coalesce(ctx.requested_server_version, "LATEST")
```

and then further down change the root url to 

```diff
-                    ("https://download.jboss.org/jbosstools/vscode/stable/lemminx-binary/%s/%s.zip"):format(
+                    ("https://download.jboss.org/jbosstools/vscode/snapshots/lemminx-binary/%s/%s.zip"):format(
```

Reply:
I don't have enough experience with lemminx to determine how stable their snapshots tend to be... but if they tend to be stable I assume LATEST would be easier to avoid having to update on every release

Reply:
Yeah pinning versions here is avoided as much as possible (I think it's only done in one other server). It's still 0-version so I guess stability is a subjective matter.

Reply:
I leave it up to you to do as you like, I don't have any strong preferences here.

Reply:
Let's go with using the snapshots distribution channel with LATEST as default

Reply:
I have update the PR to use LATEST